### PR TITLE
Cold storage 

### PR DESCRIPTION
**Pull request by caseykennedy**
This PR integrates Ledger cold storage and merges with latest master branch.

### CODE REVIEW

File: src/ui/components/AppHeader/app-header.scss [hunk 1]
```diff
@@ -79,35 +78,32 @@
       width: 0;
       height: 0;
       bottom: 100%;
-      left: 1.5em; // offset should move with padding of parent
-      border: .375rem solid transparent;
+      left: 1.5em;
+      //offsetshouldmovewithpaddingofparentborder: .375rem solid transparent;
```
Comment (rithvikvibhu) [inline src/ui/components/AppHeader/app-header.scss:82]: Looks like the comment was moved into the next property and commented out the border xD

File: src/ui/pages/SendTx/index.tsx [hunk 2]
```diff
@@ -48,44 +55,50 @@ export default function SendTx(): ReactElement {
...
-          address: 'hs1q7q3h4chglps004u3yn79z0cp9ed24rfr5ka9n5',
-        }],
-      }
+        outputs: [
+          {
+            value: +toDollaryDoos(0.1),
+            address: "hs1q7q3h4chglps004u3yn79z0cp9ed24rfr5ka9n5",
+          },
+        ],
+      },
```
Comment (rithvikvibhu) [inline src/ui/pages/SendTx/index.tsx:64]: I know it's not a change made in this PR, but any idea what this hard-coded address is, and who owns it?

File: package.json [hunk 3]
```diff
@@ -79,25 +80,30 @@
...
         "bignumber.js": "^9.0.1",
         "bootstrap": "^4.5.2",
+        "bsert": "0.0.10",
         "bufio": "^1.0.7",
+        "chrome-net": "^3.3.4",
         "classnames": "^2.2.6",
         "copy-to-clipboard": "^3.3.1",
         "eventemitter2": "^6.4.4",
         "fast-deep-equal": "^3.1.3",
+        "flatted": "^3.2.2",
```
Comment (rithvikvibhu) [inline package.json:103]: Are these 3 dependencies needed? Couldn't find them used in the code anywhere:
```md
- bsert
- chrome-net
- flatted
```

Comment (caseykennedy) [reply package.json:103]: Thank you for pointing out! Overlooked in cleanup, will address.

File: src/ui/components/AppHeader/app-header.scss (hunk 1 above)
Comment (caseykennedy) [reply src/ui/components/AppHeader/app-header.scss:82]: Thanks for pointing out, will address before merge. Thanks!

File: src/ui/pages/SendTx/index.tsx (hunk 2 above)
Comment (caseykennedy) [reply src/ui/pages/SendTx/index.tsx:64]: No idea...

**Comment by caseykennedy** [comment]
Closing as this was included in a separate branch.

**Pull request closed without merging**